lib/block_view.go

```diff
@@ -453,6 +466,16 @@ func (bav *UtxoView) CopyUtxoView() (*UtxoView, error) {
 		newView.TransactorNonceMapKeyToTransactorNonceEntry[entryKey] = &newEntry
 	}
 
+	// Copy the ValidatorEntries
+	newView.ValidatorMapKeyToValidatorEntry = make(map[ValidatorMapKey]*ValidatorEntry, len(bav.ValidatorMapKeyToValidatorEntry))
+	for entryKey, entry := range bav.ValidatorMapKeyToValidatorEntry {
+		newEntry := *entry
+		newView.ValidatorMapKeyToValidatorEntry[entryKey] = &newEntry
```

Comment:
should we just use `newEntry.Copy()`? realizing I missed that in the TransactorNonceMapKeyToTransactorNonceEntry as well :facepalm: